src/panoptes/pocs/camera/camera.py

```diff
@@ -418,6 +444,21 @@ def take_exposure(self,
 
         assert filename is not None, self.logger.error("Must pass filename for take_exposure")
 
+        if not self.can_take_internal_darks:
+            if dark and self.filterwheel and self.filterwheel.dark_position:
```

Comment:
Right, so we could have several different implementations for "prepare camera to take dark exposures", and a mechanism to select which one should be used for a given set of hardware, so that the base class only needs to make a generic method call instead of covering all possible hardware configurations itself?